scripts/fix-tslint.ts

```diff
@@ -2,44 +2,46 @@
 
 /// <reference types="node" />
 
-import * as fs from "fs";
-import * as path from "path";
-import JSON = require("comment-json");
+import * as fs from 'node:fs';
+import JSON from 'comment-json';
 
-const home = path.join(__dirname, "..", "types");
+const home = new URL('../types/', import.meta.url);
```

Comment:
why change from `path.join` to `new URL`?

Reply:
`import.meta.url` is a URL, which `fs.readdirSync()` etc. accept but `path.join()` doesn't.